repositories.yaml

```diff
@@ -2068,3 +2068,23 @@ applications:
     channels:
       - v1_name: subscription-platform-backend
         app_id: subscription.platform.backend
+
+  - app_name: experimenter_cirrus
+    canonical_app_name: "Experimenter (Cirrus)"
+    app_description: >
+      Instances of Cirrus (Mozilla's web experimentation tooling) for Experimenter (Mozilla's experimentation management application)
+    url: https://github.com/mozilla/experimenter
+    notification_emails:
+      - project-nimbus@mozilla.com
+    branch: main
+    metrics_files: []
+    ping_files: []
```

Comment:
You already have metrics and pings yamls in the repo, so let's add them here.

Reply:
Experimenter itself does not have its own metrics/pings, at this time it only uses Cirrus